### PR TITLE
Fix changing font size in text editor

### PR DESCRIPTION
**Pull request by FoSuCloud**
## References

Fixes: https://github.com/jupyterlab/jupyterlab/issues/16182
## Code changes

packages/fileeditor-extension/src/commands.ts
截屏2024-04-28 21 47 53

## User-facing changes

no
## Backwards-incompatible changes

no

### CODE REVIEW

File: galata/test/jupyterlab/texteditor.test.ts [hunk 1]
```diff
@@ -103,4 +103,42 @@ ut elit.`
     const tabHandle = await page.activity.getPanelLocator(DEFAULT_NAME);
     expect(await tabHandle?.screenshot()).toMatchSnapshot(imageName);
   });
+
+  test.describe('Changing a text editor font-size', () => {
```
Comment (fcollonval) [inline galata/test/jupyterlab/texteditor.test.ts:107]: Thanks a lot for adding a test for this :heart:

**Review by fcollonval — APPROVED** [review]
Thanks @FoSuCloud 

I spot a small glitch that the gutter line number is not updated right away when changing the font size; example when increasing the font size

image

CodeMirror does update the font size of the gutter if I switch back and forth to fileeditor. I guess forcing an update after update the font size may smooth the user experience.

**Comment by FoSuCloud** [comment]
> Thanks @FoSuCloud
> 
> I spotted a small glitch that the gutter line number is not updated right away when changing the font size; example when increasing the font size
> 
> image
> 
> CodeMirror does update the font size of the gutter if I switch back and forth to the file editor. I guess forcing an update after updating the font size may smooth the user experience.

Thanks for knowing the problem, I will force update the number of lines after updating the font size

**Comment by FoSuCloud** [comment]
It seems that the "should Decrease Content Font Size" and "should Increase Content Font Size" test cases failing have nothing to do with the pr here

**Comment by krassowski** [comment]
@meeseeksdev please backport to 4.1.x

**Pull request merged**